turbopack/crates/turbopack-ecmascript-plugins/src/transform/directives/server_to_client_proxy.rs

```diff
@@ -33,7 +34,8 @@ pub fn create_proxy_module(transition_name: &str, target_import: &str) -> Progra
                 phase: Default::default(),
             })),
             ModuleItem::Stmt(quote!(
-                "__turbopack_export_namespace__($proxy);" as Stmt,
+                "$turbopack_export_namespace($proxy);" as Stmt,
+                turbopack_export_namespace: Expr = TURBOPACK_EXPORT_NAMESPACE.into(),
```

Comment:
I hope `.into()` doesn't make it a string expression...

Reply:
No:
```rust
impl Display for TurbopackRuntimeFunctionShortcut {
    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
        f.write_str(self.full)
    }
}

impl From<&TurbopackRuntimeFunctionShortcut> for FreeVarReference {
    fn from(val: &TurbopackRuntimeFunctionShortcut) -> Self {
        FreeVarReference::Member("__turbopack_context__".into(), val.shortcut.into())
    }
}

impl From<&TurbopackRuntimeFunctionShortcut> for Expr {
    fn from(val: &TurbopackRuntimeFunctionShortcut) -> Self {
        Expr::Member(MemberExpr {
            obj: Box::new(Expr::Ident("__turbopack_context__".into())),
            prop: MemberProp::Ident(val.shortcut.into()),
            ..Default::default()
        })
    }
}
```